python/ClusterLauncher/Job.py

```diff
@@ -9,7 +9,7 @@
   pathname = os.path.abspath(pathname)
 
 # Add the utilities/python_getpot directory
-MOOSE_DIR = os.path.abspath(os.path.join(pathname, '../../'))
+MOOSE_DIR = os.path.abspath(os.path.join(pathname, '../'))
```

Comment:
Normally when you use `os.path.join` you leave off the slashes (either forwards or back). The point of that utility is to create code agnostic of platform (Windows or Unix). It's not a big deal since we don't run on Windows but just thought I'd mention it.